### PR TITLE
add support for Brother DCP-L2520DW printer/scanner

### PR DESCRIPTION
**Pull request by jostillmanns**

### CODE REVIEW

File: srcpkgs/brother-brscan4/template [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	mylibdir="lib"
+fi
+
+do_extract() {
+	ar x ${XBPS_SRCDISTDIR}/${pkgname}-${version}/brscan4-${version}-${debpkgid}.deb
+	cd ${wrksrc}
+	tar xzpvf data.tar.gz
+}
+
+do_install(){
```
Comment (Vaelatern) [inline srcpkgs/brother-brscan4/template]: Travis wants a space between `()` and `{`

File: srcpkgs/brother-brscan4/template [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Template file for 'brother-brscan4'
+pkgname=brother-brscan4
+version=0.4.3
+revision=0
```
Comment (Vaelatern) [inline srcpkgs/brother-brscan4/template]: `revision=1` instead of `0` across these commits would pass xlint

**Comment by dominikh** [comment]
@jostillmanns please address the issues reported by xlint.

Also, do we have anyone who is familiar with our CUPS and SANE packaging that can review this package?

**Comment by Gottox** [comment]
any news? - one cup zero guys?

**Comment by the-maldridge** [comment]
I'm not entirely satisfied that this contains distributable artifacts, but from a cups standpoint, I sign off on it.

**Comment by jostillmanns** [comment]
I am unfortunately not using void-linux anymore, hence I have no interest in maintaining this.

**Comment by the-maldridge** [comment]
closed by maintainer agreement

**Pull request closed without merging**